cypress/fixtures/fireboltCalls/closedcaptions.json

```diff
@@ -420,10 +420,10 @@
         "validationJsonPath": "result",
         "content": "CLOSEDCAPTIONS_BACKGROUNDOPACITY"
     },
-    "12_FOR_BACKGROUNDOPACITY_IN_CLOSEDCAPTIONS": {
+    "DEFAULT_VALUE_FOR_BACKGROUNDOPACITY_IN_CLOSEDCAPTIONS": {
         "method": "closedcaptions.backgroundOpacity",
         "validationJsonPath": "result",
-        "content": "CLOSEDCAPTIONS_BACKGROUNDOPACITY_12"
+        "content": "NULL"
     },
```

Comment:
if the expected value is NULL, no need to add the field content. By default FCS should take NULL. FIRECERT-2044 for reference